### ISSUE TITLE
можно ли изменить .exe файлы?

### ISSUE THREAD

**Issue by Dag3r** [Priler/jarvis#122]
просто хотелось бы что бы он не big picture запускал а отдельное приложение

**Comment by Dag3r** [comment]
ну или хотя бы значение поменять там 
вместо steam::\open\bigpicture такое например steam://rungameid/570

**Comment by Fahriddin3040** [comment]
Нет. Сам собранный .exe файл никак не поменять, как мне известно.

**Comment by ReekyLeon** [comment]
Да, ты можешь заменить команды для Джарвиса, естественно сами .exe ты никак не поменяешь, но заменить их на новые пожалуйста, делается не очень трудно, если всё ещё актуально то можешь написать в тг @leon340 я объясню.
